packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -152,242 +126,273 @@ export class MgtTodo extends MgtTasksBase {
-      : isCompleted
-      ? html`
-          
-        `
-      : null;
+        <span class="TaskCalendar">${getSvg(SvgIcon.Calendar)}</span>
+        <span class="TaskDueDate">${getDateString(new Date(task.dueDateTime.dateTime))}</span>
+      `
+      : html``;
 
-    let taskDetailsTemplate = null;
     if (this.hasTemplate('task-details')) {
       taskDetailsTemplate = this.renderTemplate('task-details', context, `task-details-${task.id}`);
     } else {
-      const taskDueTemplate = task.dueDateTime
-        ? html`
-            <div class="TaskDetail TaskDue">
-              <span>Due ${getShortDateString(new Date(task.dueDateTime.dateTime))}</span>
-            </div>
-          `
-        : null;
-
       taskDetailsTemplate = html`
-        <div class="TaskTitle">
-          ${task.title}
-        </div>
-        <div class="TaskDetail TaskBucket">
-          ${this.renderBucketIcon()}
-          <span>${this._currentList.displayName}</span>
-        </div>
-        ${taskDueTemplate}
+      <div class="TaskDetails">
+        <div class="Title">${task.title}</div>
+        <div class="TaskDue">${taskDueTemplate}</div>
+        <fluent-button class="TaskDelete"
+          @click="${() => this.removeTask(task.id)}"
+          aria-label="${this.strings.deleteTaskLabel}">
+          ${getSvg(SvgIcon.Delete)}
+        </fluent-button>
+      </div>
       `;
     }
 
-    const taskOptionsTemplate =
-      !this.readOnly && !this.hideOptions
-        ? mgtHtml`
-            <div class="TaskOptions">
-              <mgt-dot-options
-                .options="${{
-                  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-                  [this.strings.removeTaskSubtitle]: e => this.removeTask(e, task.id)
-                }}"
-              ></mgt-dot-options>
-            </div>
-          `
-        : null;
-
     const taskClasses = classMap({
-      Complete: isCompleted,
-      Incomplete: !isCompleted,
+      Complete: true,
       ReadOnly: this.readOnly,
       Task: true
     });
-    const taskCheckContainerClasses = classMap({
-      Complete: isCompleted,
-      Incomplete: !isCompleted,
-      TaskCheckContainer: true
-    });
+
+    const taskCheckContent = html`${getSvg(SvgIcon.CheckMark)}`;
 
     return html`
-      <div class=${taskClasses}>
-        <div class="TaskContent" @click="${(e: Event) => this.handleTaskClick(e, task)}}">
-          <span class=${taskCheckContainerClasses} @click="${(e: Event) => this.handleTaskCheckClick(e, task)}">
-            <span class=${classMap(taskCheckClasses)}>
-              <span class="TaskCheckContent">${taskCheckContent}</span>
-            </span>
-          </span>
-          <div class="TaskDetailsContainer ${this.mediaQuery}">
-            ${taskDetailsTemplate}
-          </div>
-          ${taskOptionsTemplate}
-          <div class="Divider"></div>
+      <fluent-checkbox id=${task.id} class=${taskClasses} ?checked=${true} @click="${() =>
```

Comment:
```suggestion
      <fluent-checkbox id=${task.id} class=${taskClasses} checked @click="${() =>
```

Since you're passing in `true` this will always be `checked`.